### ISSUE TITLE
Write EPUB2

### ISSUE THREAD

**Issue by BlackRobotMX** [aerkalov/ebooklib#225]
I can create EPUB3 files from HTML, is there a way for the output to be EPUB2?

**Comment by aerkalov** [comment]
You can try `epub.write_epub('test.epub', book, {"epub3_pages": False, "epub3_landmark": False, "epub2_guide": True})`.

I am just looking at the code which was doing that (and million other things) and I think that might be it.

**Comment by BlackRobotMX** [comment]
I did try writing the epub with those options, bit still when I validate the epub i get that the version is 3.0.1. 

![Captura de Pantalla 2021-02-16 a la(s) 10 29 49](https://user-images.githubusercontent.com/13722365/108092041-e9a4aa00-7041-11eb-8fc4-ae5efbf98bdb.png)

**Comment by aerkalov** [comment]
I checked the source code of epubcheck to see how it gets version of the EPUB because I forgot.  So yes, this is hard coded into the code. Line 1086 in epub.py. But you also need to use those options. 

```
        package_attributes = {'xmlns': NAMESPACES['OPF'],
                              'unique-identifier': self.book.IDENTIFIER_ID,
                              'version': '3.0'}
```

When file is created epubcheck will complain because of the different format for the metadata in the content.opf file. Plus will complain about html formatting, because template for creating pages is created for 3.0. Will take a look at it.

**Comment by BlackRobotMX** [comment]
Yes, I tried changing the version depending on the epub2_guide value, but as you say i get a lot of errors while validating the EPUB.

![Captura de Pantalla 2021-02-17 a la(s) 9 36 47](https://user-images.githubusercontent.com/13722365/108227582-aa3e9200-7103-11eb-9361-d108ef2a822c.png)
![Captura de Pantalla 2021-02-17 a la(s) 9 37 05](https://user-images.githubusercontent.com/13722365/108227637-b4609080-7103-11eb-98a8-489809817507.png)
![Captura de Pantalla 2021-02-17 a la(s) 9 37 25](https://user-images.githubusercontent.com/13722365/108227677-bfb3bc00-7103-11eb-9386-275dabeafba9.png)
![Captura de Pantalla 2021-02-17 a la(s) 9 37 38](https://user-images.githubusercontent.com/13722365/108227708-c7736080-7103-11eb-8b77-3c5bd0f3ea55.png)
